DRCODEOWNERS

```diff
@@ -19,26 +18,24 @@
 # Default owners for everything in the repo.
 # Unless a later match takes precedence, these groups will be requested for
 # review when someone opens a pull request.
-/                                                                   @datarobot/custom-models
-/custom_model_runner                                                @datarobot/custom-models @datarobot/core-modeling
-/custom_model_runner/datarobot_drum/drum/custom_fit_wrapper.py      @datarobot/core-modeling
```

Comment:
It looks like this file is no long present in repo and test requested to remove this note.